package.json

```diff
@@ -7,12 +7,12 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "prettier --config .prettierrc \"./**/*.{vue,js}\" --write"
   },
   "dependencies": {
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.0.1",
-    "core-js": "^2.6.5",
```

Comment:
Was anything using this? `vue-cli` added this as a dependency when creating the project.